streamlit_calendar/frontend/src/components/Calendar.tsx

```diff
@@ -0,0 +1,163 @@
+import "../styles/Calendar.css"
+
+import adaptivePlugin from "@fullcalendar/adaptive" // premium
+import dayGridPlugin from "@fullcalendar/daygrid"
+import interactionPlugin, { DateClickArg } from "@fullcalendar/interaction"
+import listPlugin from "@fullcalendar/list"
+import multiMonthPlugin from "@fullcalendar/multimonth"
+import FullCalendar from "@fullcalendar/react"
+import resourceDayGridPlugin from "@fullcalendar/resource-daygrid" // premium
+import resourceTimeGridPlugin from "@fullcalendar/resource-timegrid" // premium
+import resourceTimelinePlugin from "@fullcalendar/resource-timeline" // premium
+import timeGridPlugin from "@fullcalendar/timegrid"
+import timelinePlugin from "@fullcalendar/timeline" // premium
+
+import {
+  CalendarOptions,
+  EventApi,
+  EventChangeArg,
+  EventClickArg,
+  EventSourceInput,
+  ViewApi,
+} from "@fullcalendar/core"
+import React from "react"
+import { Streamlit, withStreamlitConnection } from "streamlit-component-lib"
+import { ComponentProps } from "streamlit-component-lib/dist/StreamlitReact"
+import styled from "styled-components"
+import {
+  Callback,
+  DateClickComponentValue,
+  DateClickValue,
+  EventChangeComponentValue,
+  EventChangeValue,
+  EventClickComponentValue,
+  EventClickValue,
+  EventsSetComponentValue,
+  EventsSetValue,
+  ViewValue,
+} from "../types/Calendar.type"
+
+const ENABLED_PLUGINS = [
+  adaptivePlugin,
+  dayGridPlugin,
+  interactionPlugin,
+  listPlugin,
+  multiMonthPlugin,
+  resourceDayGridPlugin,
+  resourceTimeGridPlugin,
+  resourceTimelinePlugin,
+  timeGridPlugin,
+  timelinePlugin,
+]
+
+const FullCalendarWrapper = styled.div<{ $customCSS?: string }>`
+  ${(props) => props.$customCSS}
+`
+
+type Props = ComponentProps<{
+  events?: EventSourceInput
+  options?: CalendarOptions
+  customCSS?: string
+  callbacks?: Callback[]
+  licenseKey?: string
+}>
+
+const CalendarFC: React.FC<Props> = ({
+  args: { events, options, customCSS, callbacks, licenseKey },
```

Comment:
not sure why but it seems several properties like `customCSS`  and `licenseKey` are not properly handled with this one

(note the custom CSS is not applied and a warning about license key appears)
![image](https://github.com/im-perativa/streamlit-calendar/assets/48401306/125aba0b-335c-4f79-8d3e-0ccaaf378fd2)

Reply:
Oops! you're right. Sorry, I mistakenly defined the props of the component `customCSS` and `licenseKey` instead of the correct ones, `custom_css` and `license_key` respectively. Please take a look at my fix: b46b5ae